### ISSUE TITLE
Default notifications from moderated accounts to "Ignore"

### ISSUE THREAD

**Issue by Sciasm** [mastodon/mastodon#32524]
### Pitch

Since the 4.3.0 update, notifications from moderated (limited) accounts appear in a "Filtered notifications" section instead of being dropped completely. This is not clear to some users, resulting in re-reports of already moderated accounts and servers, thus creating an additional workload for admins and moderation teams.

Please consider defaulting the notification setting for moderated accounts to "Ignore" rather than "Filter".

### Motivation

With the ongoing spam wave, some server admins and their moderation teams are receiving dozens of recurring reports for accounts and servers that have already been limited. It is becoming apparent that the meaning of the filtered notifications from moderated accounts is not clear to some users, resulting in them thinking of the notifications as unmoderated. As a result, the moderated accounts are re-reported several times, as the reporters seem to expecting them to be gone from their filtered notifications after being limited by the moderation team.

This creates an unnecessary additional workload for the (mostly) voluntary moderation teams, as they try to navigate the reports and figure out which ones are legit and which ones are duplicates. It also adds a challenge in terms of community management, as users who think of the filtered notifications from limited accounts as unmoderated, need to be made aware that the moderation team is, in fact, not neglecting their job.

This changes the moderation workflow, and as a side effect, results in suspensions where previously limits would have sufficed.

As a server admin, I kindly ask you to consider switching the default setting for notifications from moderated accounts to "Ignore" in order to avoid comfusion among users and reduce the burden on admins and moderation teams.

Thank you so much!

**Comment by DasMetalKitty** [comment]
I totally agree with this. I believe it would help correct the balance between transparency for users about notifications from limited servers, clear indication that moderation actions have been taken and the user does not need to report the spam, and keeping admin/moderation workloads manageable during a spam wave.

**Comment by ThomasLeister** [comment]
I agree! Although notifications are hidden / filtered, there will be this banner which hides those notifications. I guess by human nature it will cause users to click the "Filtered notifications" button and maybe makes them want to get rid of these notifications by reporting (already limited) spammers again. 

It makes sense to me to ignore notifications from limited servers by default.
